src/command/game/index.js

```diff
@@ -69,13 +40,18 @@ const getGameWithOptionalFilter = async (games, option) => {
         ) !== -1
     );
 
-    if (!potentialGames.length)
+    console.log({ potentialGames });
```

Comment:
@xxhomey19 Is this here because you forgot to remove it or is this done on purpose?

Reply:
oops, I forgot to remove it. thanks